app/src/main/res/layout/activity_profile.xml

```diff
@@ -68,17 +113,31 @@
 
     <EditText
```

Comment:
Cirrus complains that autofillHints is missing. Would be nice if that was added, even with some null solutions.

Reply:
Added autofillHints!